modules/op-release-notes-1-5.adoc

```diff
@@ -0,0 +1,317 @@
+
+[id="new-features-1-5_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.5.
+
+* Pipeline run and task runs will be automatically pruned by a cron job in the target namespace. The cron job uses the `IMAGE_JOB_PRUNER_TKN` environment variable to get the value of `tkn image`. With this enhancement, the following fields are introduced to the `TektonConfig` custom resource:
++
+[source,yaml,subs="attributes+"]
+----
+...
+pruner:
+  resources:
+    - pipelinerun
+    - taskrun
+  schedule: "*/5 * * * *" // cron schedule
+  keep: 2 // delete all keeping n
+...
+----
+
+* In {product-title}, you can customize the installation of the Tekton Add-ons component by modifying the values of the new parameters `clusterTasks` and `pipelinesTemplates` in the `TektonConig` custom resource.
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  profile: all
+  targetNamespace: openshift-pipelines
+  addon:
+    params:
+    - name: clusterTasks
+      value: "true"
+    - name: pipelineTemplates
+      value: "true"
+...
+----
++
+The customization is allowed if you create the add-on using `TektonConfig`, or directly by using Tekton Add-ons. However, if the parameters are not passed, the controller will add parameters with default values.
++
+[NOTE]
+====
+* If add-on is created using the `TektonConfig` custom resource, and you change the parameter values later in the
+`Addon` custom resource, then the changes will be overwritten by the values in the `TektonConfig` custom resource.
+
+* The value of the `pipelinesTemplates` parameter can be `true` only when the value of the `clusterTasks` parameter is `true`.
+====
+
+* The `enableMetrics` parameter is added to the `TektonConfig` custom resource. You can use it to disable the service monitor, which is part of Tekton Pipelines for {product-title}.
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  profile: all
+  targetNamespace: openshift-pipelines
+  pipeline:
+    params:
+    - name: enableMetrics
+      value: "true"
+...
+----
+
+* Eventlistener OpenCensus metrics, which captures metrics at process level, has been added.
+
+* Triggers now has label selector; you can configure triggers for an event listener using labels.
+
+* The `ClusterInterceptor` custom resource definition for registering interceptors is added, which allows you to register new `Interceptor` types that you can plug in. In addition, the following relevant changes are made:
+
+** In the trigger specifications, you can configure interceptors using a new API that includes a `ref` field to refer to a cluster interceptor. In addition, you can use the `params` field to add parameters that will be passed on to the interceptors for processing.
```

Comment:
to avoid future tense, consider rephrasing to, `...add parameters that pass on to the interceptors for processing.`